bin/install_ipython.sh

```diff
@@ -92,6 +86,225 @@ if [ ! -d "/etc/skel/.ipython/nbextensions" ] ; then
+
+
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-affine_1.2.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-click_4.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-snuggs_1.3.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-cligj_0.2.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-fiona_1.6.0pre0_$ARCH.deb"
+
+dpkg -i python2-affine_1.2.0_all.deb
+dpkg -i python2-click_4.0_all.deb
+dpkg -i python2-snuggs_1.3.1_all.deb
+dpkg -i python2-cligj_0.2.0_all.deb
+dpkg -i python2-fiona_1.6.0pre0_$ARCH.deb
+
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-affine_1.2.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-click_4.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-snuggs_1.3.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-cligj_0.2.0_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-fiona_1.6.0pre0_$ARCH.deb"
+
+dpkg -i python3-affine_1.2.0_all.deb
+dpkg -i python3-click_4.0_all.deb
+dpkg -i python3-snuggs_1.3.1_all.deb
+dpkg -i python3-cligj_0.2.0_all.deb
+dpkg -i python3-fiona_1.6.0pre0_$ARCH.deb
+
+
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-rasterio_0.24.1_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-netcdf4_1.1.9_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-descartes_1.0.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-geojson_1.2.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python2-geopandas_0.1.1_all.deb"
+
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-rasterio_0.24.1_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-netcdf4_1.1.9_$ARCH.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-descartes_1.0.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-geojson_1.2.1_all.deb"
+wget -c --progress=dot:mega "http://download.osgeo.org/livedvd/data/ipython/debs/Jupyter-debs/python3-geopandas_0.1.1_all.deb"
+
+
+
+dpkg -i python2-rasterio_0.24.1_$ARCH.deb
```

Comment:
Those packages (netcdf, descartes, geojson, geopandas,rasterio) already exist in OSGeoLive ppa. Can you please use those instead?